src/components/FilterBar.jsx

```diff
@@ -24,9 +19,7 @@ export default function FilterBar({ setQueryParam, queryParams }) {
       justifyContent="space-between"
       sx={{ flexGrow: 1 }}>
       <Grid item>
-        {experiments.expShowMajorFilter && (
-          <FilterDropDown label="Major" items={majors} />
```

Comment:
Hmm we're not using this component anymore. I wonder if there's any value in keeping it.

Reply:
removed it.